src/server/scripts/EasternKingdoms/ZulGurub/boss_mandokir.cpp

```diff
@@ -302,11 +399,48 @@ class npc_ohgan : public CreatureScript
 
         void Reset() override
         {
-            SunderArmor_Timer = 5000;
+            SunderArmor_Timer = urand(6000, 12000);
```

Comment:
No, this is old. Use either TaskScheduler or EventMap.

Reply:
pending

Reply:
Done